recipes/mmseqs2/meta.yaml

```diff
@@ -39,3 +39,12 @@ about:
   home: https://github.com/soedinglab/mmseqs2
   license: GPLv3
   summary: "MMseqs2.0: ultra fast and sensitive sequence search and clustering suite"
+
+extra:
+  identifiers:
+    - doi:10.1038/nbt.3988
+    - doi:10.1038/s41467-018-04964-5
+    - doi:10.1093/bioinformatics/bty1057
+    - doi:10.1101/2020.11.27.401018
+    - biotools:mmseqs2
+    - biotools:linclust
```

Comment:
how can one tool have two bio.tools IDs?

Reply:
It's two separate publications, but under the codebase/repository. Not sure if it was a good idea to make.

Reply:
Ok, that is new. We can leave it like it is.